src/main/java/uk/gov/hmcts/reform/sendletter/tasks/SerialTaskRunner.java

```diff
@@ -55,6 +58,11 @@ public void tryRun(int id, Runnable runnable) throws SQLException {
                     }
                 }
             }
+        } catch (SQLException exception) {
+            log.error(
+                String.format("SQL error occurred during task %s run", task.name()),
+                new TaskRunnerException(exception)
```

Comment:
I think it would be better to throw the new runtime exception you've created here instead of 'silently' failing. The whatever is calling this would handle exception in a desired way.

In the spirit of 'throw early, catch late' :)